modules/src/core/ics02_client/client_state.rs

```diff
@@ -35,6 +35,9 @@ pub trait ClientState: Clone + core::fmt::Debug + Send + Sync {
     /// Freeze status of the client
     fn is_frozen(&self) -> bool;
```

Comment:
Now that we have `frozen_height()` method in the `ClientDef` trait, I think we can provide a default `is_frozen()` implementation. 
```suggestion
    fn is_frozen(&self) -> {
        self.frozen_height.is_some()
    }
```